portfolio/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -1,12 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-  <application>YOUR_PROJECT_ID_HERE</application>
-  <version>1</version>
   <threadsafe>false</threadsafe>
   <sessions-enabled>true</sessions-enabled>
   <runtime>java8</runtime>
   <static-files>
     <!-- prevent unwanted caching when accessing via the web preview server -->
     <include path="/**" expiration="0s" />
   </static-files>
-</appengine-web-app>
+</appengine-web-app>
```

Comment:
Let's try to have newline at the end of files.

Reference:
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

Here and below.

Reply:
Done.